x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlFeatures.java

```diff
@@ -12,10 +12,18 @@
 import org.elasticsearch.features.NodeFeature;
 
 import java.util.Map;
+import java.util.Set;
 
 public class EsqlFeatures implements FeatureSpecification {
+    private static final NodeFeature MV = new NodeFeature("esql.mv");
+
+    @Override
+    public Set<NodeFeature> getFeatures() {
+        return Set.of(MV);
```

Comment:
A feature should be declared either as regular, or historical.
if it is historical, and used in tests only, you may consider adding it to `RestTestLegacyFeatures` instead.

Reply:
I see. I'll remove it from this line and get it working again.